docs/source/conf.py

```diff
@@ -207,6 +207,7 @@
     'user': ('https://github.com/%s', '@')
 }
 
+#autoclass_content = 'both'
```

Comment:
Concatenates both the class and init method's docstrings. Wanted to know if anyone had a particular opinion on whether this should be included or not!

Reply:
Do you have examples on what this looks like in the docs?